client/app/pages/queries/visualization-embed.js

```diff
@@ -33,7 +33,8 @@ export default function (ngModule) {
     return session($http, $route, Auth).then(() => {
       const queryId = $route.current.params.queryId;
       const query = $http.get(`/api/queries/${queryId}`).then(response => response.data);
-      const queryResult = $http.get(`/api/queries/${queryId}/results.json`).then(response => response.data);
+      // pass down any params from the top level url
+      const queryResult = $http.post(`/api/queries/${queryId}/results.json${location.search}`).then(response => response.data);
```

Comment:
Let's keep the old behavior and only use the new one, if it's enabled via [the feature flag](https://github.com/getredash/redash/blob/master/redash/settings.py#L236).